cg/cli/upload/scout.py

```diff
@@ -56,4 +58,46 @@ def scout(context, re_upload, print_console, case_id):
         )
         context.abort()
 
-    scout_api.upload(scout_config, force=re_upload)
+    context.invoke(upload_case_to_scout, case_id=case_id, re_upload=re_upload)
+
+
+@click.command()
+@click.option("-r", "--re-upload", is_flag=True, help="re-upload existing analysis")
+@click.option("--dry-run", is_flag=True)
+@click.argument("case_id")
+@click.pass_context
+def upload_case_to_scout(context, re_upload, dry_run, case_id):
+    """Upload variants and case from analysis to Scout."""
+
+    click.echo(click.style("----------------- CONFIG -----------------------"))
+
+    def _get_load_config_from_hk(hk_api: hk.HousekeeperAPI, case_id):
+        tag_name = "scout-load-config"
+        version_obj = hk_api.last_version(case_id)
+        scout_config_files = hk_api.get_files(
```

Comment:
Just curious - why does one expect to get back multiple scout configs per case bundle? Should there not only be one in the last version bundle?

Reply:
It's just because the function returns a iterable with files. It is probably one file but it is correct to name it files as it is not a single entity